packages/Interface/src/DTK_C_API.cpp

```diff
@@ -97,76 +97,26 @@ void PolyhedronListDataFunctionWrapper(
              is_ghost_cell.data() );
 }
 
-void CellListSizeFunctioWrapper( std::shared_ptr<void> user_data,
```

Comment:
How did that typo make it through?  It tool me a while to understand what changed :D